source/Depreciation.cpp

```diff
@@ -63,6 +63,41 @@ double Depreciation::Full()
 
 
 
+// Calculate the value fraction for an item of the given age.
+double Depreciation::Depreciate(int age)
+{
+	if(age <= GracePeriod())
+		return 1.;
+
+	if(age >= MaxAge())
+		return Min();
+
+	double daily = pow(Daily(), age - GracePeriod());
+	double linear = double(MaxAge() - age) / (MaxAge() - GracePeriod());
```

Comment:
why?
```suggestion
	double linear = static_cast<double>(MaxAge() - age) / (MaxAge() - GracePeriod());
```

Reply:
Because that's what I thought Hurleveur meant about removing static?